doc/api/magic-profile.md

```diff
@@ -0,0 +1,98 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Documentation for the %sqlcmd profile
+    keywords: jupyter, sql, jupysql, sqlcmd, profile
+    property=og:locale: en_US
+---
+
+# `%sqlcmd profile`
+
++++
+
+## Load Data
+
+```{code-cell} ipython3
+%load_ext sql
+%sql duckdb://
+```
+
+```{code-cell} ipython3
+from pathlib import Path
+from urllib.request import urlretrieve
+
+if not Path("penguins.csv").is_file():
+    urlretrieve(
+        "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv",
+        "penguins.csv",
+    )
+```
+
+```{code-cell} ipython3
+:tags: [hide-output]
```

Comment:
```suggestion
```

Let's not hide the output since it's useful for the user. Usually, we hide it when it's very long and distracting